label_sync/main.go

```diff
@@ -646,17 +701,53 @@ type labelData struct {
 }
 
 func writeDocs(template string, output string, config Configuration) error {
-	data := []labelData{
-		{"both issues and PRs", "both-issues-and-prs", config.LabelsByTarget(bothTarget)},
-		{"only issues", "only-issues", config.LabelsByTarget(issueTarget)},
-		{"only PRs", "only-prs", config.LabelsByTarget(prTarget)},
+	var desc string
+	data := []labelData{}
+	desc = "all repos, for both issues and PRs"
```

Comment:
How does GitHub stop us from adding labels to an issue or PR? I thought they were all global